src/hooks/index.js

```diff
@@ -1,5 +1,6 @@
 export { default as useAreas } from "./useAreas";
 export { default as useCanvas } from "./useCanvas";
+export { default as useFullscreen } from "./useFullscreen";
```

Comment:
Where was this defined? I can't seem to find it anywhere

Reply:
I rebased twice and lost the file while resolving conflicts.
Thanks, fixed